bentoml/_internal/cli/model_management.py

```diff
@@ -172,37 +176,47 @@ def delete_target(target):
 
                 if delete_confirmed:
                     model_store.delete(model.tag)
-                    click.echo(f"{model} deleted")
+                    logger.info(f"{model} deleted")
 
         for target in delete_targets:
             delete_target(target)
 
-    @model_cli.command(help="Export Model to a tar file")
+    @model_cli.command()
     @click.argument("model_tag", type=click.STRING)
-    @click.argument(
-        "out_file", type=click.File("wb"), default=sys.stdout, required=False
-    )
-    def export(model_tag, out_file):
-        """Export Model files to a tar file
+    @click.argument("out_path", type=click.STRING, default="", required=False)
+    def export(model_tag, out_path):
+        """Export Model files to an archive file
+
+        arguments:
+
+        \b
+        MODEL_TAG: model identifier
+        OUT_PATH: output path of exported model.
+          If this argument is not provided, model is exported to name-version.bentomodel in the current directory.
+          Supported formats are tar('tar'), tar.gz ('gz'), tar.xz ('xz'), tar.bz2 ('bz2'), and zip
 
+        examples:
+
+        \b
+        bentoml models export FraudDetector:latest > my_model.tar
         bentoml models export FraudDetector:latest > my_model.tar
         bentoml models export FraudDetector:20210709_DE14C9 ./my_model.tar
+        bentoml models export FraudDetector:20210709_DE14C9 s3://mybucket/models/my_model.gz
```

Comment:
similarly, let's use `.bentomodel` extension here as the default.  We can mention it also supports `.tar` or `.gz`, but always recommend users to use `.bento` or `.bentomodel`, this is because it makes it obvious what it is during `import`, user would know immediately which API to use, `bentoml models import` or `bentoml import`.